GoogleForCreators/web-stories-wp
# Redesign: Animation panel

## Summary
Redesigns the animation panel


## To-do

- [ ] Verify tests all work.



## User-facing changes
| Before | After |
|--------|------|
| <img width="301" alt="none-old" src="https://user-images.githubusercontent.com/3294597/111654249-dfc0c300-8808-11eb-8931-16a19b8db196.png"> | <img width="300" alt="none" src="https://user-images.githubusercontent.com/3294597/111653482-3f6a9e80-8808-11eb-9522-8a34e323bf5d.png"> |
| <img width="298" alt="Screenshot 2021-03-18 at 17 47 22" src="https://user-images.githubusercontent.com/3294597/111655114-a9377800-8809-11eb-8074-ed19f1b91536.png"> | <img width="298" alt="Screenshot 2021-03-18 at 17 28 34" src="https://user-images.githubusercontent.com/3294597/111653519-485b7000-8808-11eb-8300-4aa185fb38eb.png"> |
| <img width="300" alt="Screenshot 2021-03-18 at 17 48 08" src="https://user-images.githubusercontent.com/3294597/111655222-bf453880-8809-11eb-9a6c-7dc6921936fc.png"> | <img width="300" alt="Screenshot 2021-03-18 at 17 28 46" src="https://user-images.githubusercontent.com/3294597/111653548-4e515100-8808-11eb-8682-119bd51cd08b.png"> |
| <img width="301" alt="Screenshot 2021-03-18 at 17 50 36" src="https://user-images.githubusercontent.com/3294597/111655639-1c40ee80-880a-11eb-9b70-c4bd1af07c84.png"> | <img width="298" alt="Screenshot 2021-03-18 at 17 29 19" src="https://user-images.githubusercontent.com/3294597/111653584-53ae9b80-8808-11eb-859a-a9fba24cb813.png"> |
| <img width="307" alt="Screenshot 2021-03-18 at 17 45 09" src="https://user-images.githubusercontent.com/3294597/111654774-59f14780-8809-11eb-8735-cce922025bc3.png"> | <img width="281" alt="Screenshot 2021-03-18 at 17 36 47" src="https://user-images.githubusercontent.com/3294597/111653672-60cb8a80-8808-11eb-865e-6a45cba1f92f.png"> |
| <img width="301" alt="Screenshot 2021-03-18 at 17 50 43" src="https://user-images.githubusercontent.com/3294597/111655679-2367fc80-880a-11eb-9a25-53e0e6bcd13b.png"> | <img width="299" alt="Screenshot 2021-03-18 at 17 37 01" src="https://user-images.githubusercontent.com/3294597/111653734-6a54f280-8808-11eb-99fb-b6fdf161ea12.png"> |


### QA






This PR can be tested by following these steps:

1. Open a shape
2. Verify the Animations panel looks like in Figma. Try out with different animation types.
3. Now assign media to the background and open the animation panel for background media
4. Verify everything looks as per Figma. Please try "Pan & Zoom" effect since it's using a dropdown as one of the inputs.
See [Figma.](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=1923%3A424212)

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6505
